cp "-t" option does not exists under FreeBSD cp

Teetoow opened the pull request:
It's a new GNU cp option added in order to use cp with xargs. If xargs is not used this option is useless since the output directory have to be specified as the last argument (by default).

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ONLYOFFICE/sdkjs?pullRequest=1099) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/ONLYOFFICE/sdkjs?pullRequest=1099) before we can accept your contribution.<br/>**9** out of **10** committers have signed the CLA.<br/><br/>:white_check_mark: K0R0L<br/>:white_check_mark: KhromovNikita<br/>:white_check_mark: KirillovIlya<br/>:white_check_mark: SergeyLuzyanin<br/>:white_check_mark: ayuzhin<br/>:white_check_mark: GoshaZotov<br/>:white_check_mark: catbasilio<br/>:white_check_mark: Teetoow<br/>:white_check_mark: konovalovsergey<br/>:x: Antipin Nikita<br/><hr/>**Antipin Nikita** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ONLYOFFICE/sdkjs?pullRequest=1099) it.</sub>

K0R0L committed:
New classes for themes map (#1784)

Co-authored-by: Nikita Khromov <Nikita.Khromov@onlyoffice.com>

KirillovIlya committed:
Fix bug #49443

Fix the problem with compiling table cell properties

GoshaZotov committed:
[se] Modify RecalculateAllFormulas function

KirillovIlya committed:
Fix bug #49526

ayuzhin committed:
[native] Extend review model

ayuzhin committed:
Merge pull request #1788 from ONLYOFFICE/fix/native-review

[native] Extend review model

K0R0L committed:
Fix bug 49556 (#1790)

KhromovNikita committed:
Added theme's map for disabled select (#1791)

AlexeyMatveev686 committed:
Added theme's map for disabled form-control (#1792)

SergeyLuzyanin committed:
[bug] For bugs 49575, 49576: fix opening pptx with animation

GoshaZotov committed:
[se] Fix bug 49564

SergeyLuzyanin committed:
[bug] Fix bug 49578

KirillovIlya committed:
For bug #49135

Add ability to fill picture in special picture content control when pasting data

GoshaZotov committed:
[se] Fix bug 49135

K0R0L committed:
Add support real-time change themes in plugins (#1797)

KhromovNikita committed:
Added themes support for scroll (#1799)

K0R0L committed:
Main events in plugins (for example onDocumentContentReady) (#1800)

KirillovIlya committed:
Fix bug #49571

GoshaZotov committed:
Fix/bug 49566 (#1802)

* [se] Fix bug 49566

Commit:
[se] Fixed bugs in api builder for SE

GoshaZotov committed:
Fix/bug 49592 (#1803)

* [se] Fix bug 49592

KirillovIlya committed:
Fix bug #49590

SergeyLuzyanin committed:
[bug] Fix bug 49614

K0R0L committed:
Add generate TOC buttons (#1805)

GoshaZotov committed:
[se] Change error type

KirillovIlya committed:
Fix bug #49610

SergeyLuzyanin committed:
[bug] Fix bug 49620

konovalovsergey committed:
[de] Fix c_oAscError.ID.Submit export

Commit:
Feature/colors update (#1815)

* Color updates

* Update scroll thumb border

KhromovNikita committed:
Added support themes for select2's arrow (#1817)

konovalovsergey committed:
[se] Fix bug 49672

konovalovsergey committed:
[se] Fix bug 49532

K0R0L committed:
Change the meaning of a parameter isModal (#1818)

konovalovsergey committed:
[se] Fix bug 49639

KhromovNikita committed:
Fix scroll border's style (#1820)

SergeyLuzyanin committed:
[bug] Fix resetting chart object on legend recalculation

GoshaZotov committed:
[se] Fix bug 49693

KhromovNikita committed:
Link's styles for themes (#1821)

KhromovNikita committed:
Form control[readonly]/[disabled] styles for themes (#1822)

GoshaZotov committed:
[se] By bug 49693

KhromovNikita committed:
Added description for Api.ReplaceTextSmart() method (#1825)

SergeyLuzyanin committed:
[bug] Fix bug 49703

KirillovIlya committed:
Implement function to detect did we hit in form/content control

K0R0L committed:
Enable mouse hand mode in FillingForms mode

Commit:
Fixed bug with horizontal scrolling (#1827)

GoshaZotov committed:
[se] Fix bug 49739

KirillovIlya committed:
Fix bug #49747

KirillovIlya committed:
Fix issue with new hand mode

K0R0L committed:
Fix typo

K0R0L committed:
Unlock mouse on up event (if hand mode enable)

KirillovIlya committed:
Fix bug #49748

KirillovIlya committed:
Change the cursor type when hovering over the checkbox

KirillovIlya committed:
Remove cursor in case of checkbox/radiobutton in the filling form mode

K0R0L committed:
Fix bug 49748

K0R0L committed:
Unlock selection if it started in form/cc

KirillovIlya committed:
Fix bug #49749

KirillovIlya committed:
Now we toggle checkbox on mousedown instead of mouseup

SergeyLuzyanin committed:
[bug] Fix bug 49744

KirillovIlya committed:
Implement base functions to manage showing of adjustments

SergeyLuzyanin committed:
check isShowShapeAdjustments flag on drawing shapes tracks

KirillovIlya committed:
Fix bug #44625

K0R0L committed:
Fix/buttons cc (#1831)

* Refactoring buttons for cc

* Fix 1.5 scaling

konovalovsergey committed:
[se] Fix bug 49639

K0R0L committed:
Fix bugs with draw cc (#1833)

KirillovIlya committed:
Fix bug #49807

konovalovsergey committed:
[se] Fix bug 49700

SergeyLuzyanin committed:
Add isShowShapeAdjustments, isShowTableAdjustments, isShowEquationTrack to se and pe

SergeyLuzyanin committed:
[bug] Fix bug 49745

konovalovsergey committed:
[se] Fix bug 49682

KirillovIlya committed:
Fix bug #32521

KirillovIlya committed:
Fix bug #49810

KirillovIlya committed:
For bug #49727

Now function GetTextFormAutoWidth returns twips instead of mm

SergeyLuzyanin committed:
[bug] For bug 49733

SergeyLuzyanin committed:
Remove Type, Image properties

KirillovIlya committed:
Fix bug #49809

KirillovIlya committed:
Fix bug #49829

Cancel special editing on change editing restriction

KirillovIlya committed:
Fix bug #49831

KirillovIlya committed:
Fix bug #49837

K0R0L committed:
Fix bug with not cell scale (#1839)

KirillovIlya committed:
Improve check hit in form/content control

K0R0L committed:
Fix bug 49873, fix bug 49839 (#1844)

KirillovIlya committed:
Fix bug #49748

AlexeyMatveev686 committed:
Added styles for default scroll (#1845)

Fix bug #49766 #49771

K0R0L committed:
Fix bug 49802 (#1846)

SergeyLuzyanin committed:
[bug] Fix bug 49861; fix bug 49874

KirillovIlya committed:
Fix bug #49879

KirillovIlya committed:
Fix bug #49881

K0R0L committed:
Add method GetSelectionType for plugins (#1848)

SergeyLuzyanin committed:
[bug] Fix bug 49872

KirillovIlya committed:
Fix check hit in content control in case of shape

GoshaZotov committed:
[se] Fix bug 49896

K0R0L committed:
Fix bug 49884

SergeyLuzyanin committed:
[bug] Fix cursor updating in view mode over shapes contents

K0R0L committed:
Fix bug 49746 (#1853)

GoshaZotov committed:
[se] Show argument type after add and delete data

GoshaZotov committed:
[se] Fix export c_oAscFormulaArgumentType

konovalovsergey committed:
[de] Fix bug 49842

K0R0L committed:
Fix bug 49916 (#1857)

SergeyLuzyanin committed:
[bug] Fix bug 49910

SergeyLuzyanin committed:
[bug] Fix updating chart type in the right panel

KirillovIlya committed:
Fix bug #49899

Fix the problem with clearing image form

KirillovIlya committed:
Fix bug #49574

Fix the problem with calculating spacing before/after in case of content controls

KirillovIlya committed:
Fix bug #49926

SergeyLuzyanin committed:
[bug] Fix bug 49938

SergeyLuzyanin committed:
[bug] Fix bug 49934

SergeyLuzyanin committed:
[bug] Fix bug 49935

SergeyLuzyanin committed:
[bug] Fix bug 49957

SergeyLuzyanin committed:
[bug] Fix bug 49960

SergeyLuzyanin committed:
[bug] For bug 49177

SergeyLuzyanin committed:
[bug] Fix bug 49815

konovalovsergey committed:
[se] Fix bug 49918

SergeyLuzyanin committed:
[bug] Fix bug 49941

SergeyLuzyanin committed:
[bug] Fix bug 49971

KirillovIlya committed:
Fix bug #49988

KirillovIlya committed:
Fix bug #49972

Fix the problem with auto color for shading

KhromovNikita committed:
Themes map for scroll (#1866)

GoshaZotov committed:
[se] Add function in api (#1867)

[se] For get link bug

GoshaZotov committed:
[se] Escape special symbols(generate html)

Commit:
Fix bug 49395 (#1810)

* Fix bug 49395

* Added the correct tab button size when scaling

* Fixed button tabs size in scale

GoshaZotov committed:
Fix/add builder property (#1869)

* [se] Add properties description

GoshaZotov committed:
[se] Fix bug 49931 (#1872)

* [se] Fix bug 49931

SergeyLuzyanin committed:
[bug] For bug 50027

K0R0L committed:
Feature/mobileoverlaytable (#1871)

* Added mobile overlay table scaling

* Fix draw in pixel net

* Changed the arrow rendering method

* Change table track in mobile version

Co-authored-by: Nikita Antipin <nikita.antipin@onlyoffice.com>

SergeyLuzyanin committed:
[bug] Fix bug 50042

konovalovsergey committed:
[all] For bug 49716

K0R0L committed:
Fix bug 49460 (#1876)

K0R0L committed:
Fix bug 50000 (#1878)

SergeyLuzyanin committed:
[bug] Fix bug 50064

K0R0L committed:
Fix bug 50063 (#1879)

GoshaZotov committed:
[se] Fix bug 50100

K0R0L committed:
Fix/bug50108 (#1881)

* Fix bug 50108, 50067

* Fix tests

GoshaZotov committed:
Fix/bug 49724 (#1882)

* [se] Fix bug 49724

KirillovIlya committed:
Fix bug #50076

Prevent going outside of a form when selection was started inside this form

K0R0L committed:
Fix bug 50058 (#1885)

KirillovIlya committed:
Fix previous commit

K0R0L committed:
Fix bug with show "default" cursor in fill forms mode

konovalovsergey committed:
[de] For Bug #49807

K0R0L committed:
Fix bug 49965 (#1887)

KirillovIlya committed:
Improve selection of fields in filling forms mode

KirillovIlya committed:
Fix bug #49750

Remove cursor in filling form mode from checkboxes in shaped

KirillovIlya committed:
Prevent selection when clicking on checkbox

KirillovIlya committed:
Fix previous commit

SergeyLuzyanin committed:
[bug] Fix bug 50111

SergeyLuzyanin committed:
[bug] For bug 50111

KirillovIlya committed:
Fix click on checkox

SergeyLuzyanin committed:
[bug] For bug 49949:
change multiplier for double reading/writing from 10000 to 100000
[bug] Fix historyitem_Unknown_Unknown changes to correct change types in CAnimRot

GoshaZotov committed:
[se] Fix bug 50106

K0R0L committed:
Fix bug 49876 (#1891)

SergeyLuzyanin committed:
[bug] Fix applying changes to animation objects in coauthoring

GoshaZotov committed:
[se] Fix bug 49958

ovchinnikova-natalya committed:
builder: some changes for the property descriptions

K0R0L committed:
Fix bug 50113 (#1895)

KirillovIlya committed:
Fix bug #50080

Rework rendering tracks around forms

K0R0L committed:
Fix/bug50134 (#1896)

* Fix bug 50134

* .

SergeyLuzyanin committed:
[bug] For bug 49932: remove animation for smart-art elements to avoid broken files

KirillovIlya committed:
Improve correction of select in the filling form mode

konovalovsergey committed:
[se] Get translation of 'Qtr' and '%1 or %2' from translation manager

konovalovsergey committed:
[se] Get translation of 'Total' from translation manager

KirillovIlya committed:
Fix bug #50133

Also add correction of forms when opening file

GoshaZotov committed:
[se] Fix bug 50144

K0R0L committed:
Fix bug 50142 (#1900)

Commit:
Fix bug 49395 (#1898)

* Fix bug 49395

* Release/v6.3.0 (#1901)

* [se] Fix bug 50144

* Fix bug 50142 (#1900)

Co-authored-by: GoshaZotov <Igor.Zotov@onlyoffice.com>

Co-authored-by: Oleg Korshul <Oleg.Korshul@onlyoffice.com>
Co-authored-by: GoshaZotov <Igor.Zotov@onlyoffice.com>

KirillovIlya committed:
Fix bug #50156

KirillovIlya committed:
Fix bug #49925

konovalovsergey committed:
[se] Fix bug 50143

SergeyLuzyanin committed:
[bug] Fix bug 50176

GoshaZotov committed:
[se] Fix bug 50188

GoshaZotov committed:
[se] Fix bug 50198

konovalovsergey committed:
[se] Fix bug 49604

KhromovNikita committed:
Fix plugins link color (#1906)

* Fix plugins link color

* Added hover, active, visited link's state colors

AlexeyMatveev686 committed:
Fix bug #50184 (#1908)

SergeyLuzyanin committed:
[bug] Fix bug 50180

SergeyLuzyanin committed:
Add empty line

SergeyLuzyanin committed:
[bug] Fix bug 50204

SergeyLuzyanin committed:
[bug] Fix bug 50201

GoshaZotov committed:
[se] Fix bug 49019

GoshaZotov committed:
[se] Fix bug 50169

K0R0L committed:
Fix bug 49885 (#1912)

GoshaZotov committed:
[se] Fix bug 50214, fix bug 50215

konovalovsergey committed:
[se] Fix bug 49632

SergeyLuzyanin committed:
[bug] For bug 50201: fix stock data validation

GoshaZotov committed:
Fix/bug 49257 (#1915)

* [se] Fix bug 49257

GoshaZotov committed:
[se] For bug 50235: fix save to html

konovalovsergey committed:
[de] Fix bug with 1970 comment date after copy-paste

K0R0L committed:
Fix bug 50237 (#1918)

K0R0L committed:
Fix/colors (#1919)

* Add read theme colors

* Support rgba(..,) colors

* Fix crash on changing theme

* Update color map

* Fix smooth in vertical ruler

Co-authored-by: evgenykatyshev <me@ekatyshev.ru>

K0R0L committed:
Fix parsing colors (#1920)

K0R0L committed:
Fix bug 46682, 47661 (#1921)

SergeyLuzyanin committed:
[bug] Fix bug 50201

SergeyLuzyanin committed:
[bug] Fix bug 47923

K0R0L committed:
Fix bugs with slides skin (#1922)

Commit:
Fix bug 50240

K0R0L committed:
Fix colors in reporter mode (#1923)

Commit:
Added comments for bug

GoshaZotov committed:
[se] Fix bug 50240

K0R0L committed:
Fix bugs with themes (#1925)

GoshaZotov committed:
Fix/bug 50004 (#1926)

* [se] For bug 50004

konovalovsergey committed:
[se] Fix name containing non-ASCII symbols

GoshaZotov committed:
[se] Fix typo

papacarlo committed:
Merge branch hotfix/v6.2.2 into release/v6.3.0

K0R0L committed:
Fix bug 49018 (#1929)

K0R0L committed:
Fix bug 50299 (#1930)

KhromovNikita committed:
Fix bug #50306

K0R0L committed:
Fix bug 50335 (#1933)

Commit:
Fix bug 50279 (#1934)

K0R0L committed:
Fix bug 50321, 50320, 50322, 50333 (#1935)

KirillovIlya committed:
For bug #50004

Commit:
Fix bug #50318

GoshaZotov committed:
Fix/bug 50004 (#1938)

* [se] By bug 50004: complex field

GoshaZotov committed:
[se] Fix bug 50350

K0R0L committed:
Fix bug 50363 (#1939)

K0R0L committed:
Fix bug 50310 (#1941)

KirillovIlya committed:
Fix problem with first update of the rulers

SergeyLuzyanin committed:
[bug] Fix bug 50392

GoshaZotov committed:
[se] Fix bug 50394

SergeyLuzyanin committed:
[bug] Fix bug 50399

K0R0L committed:
Fix bug 50337 (#1946)

konovalovsergey committed:
[se] Fix unnecessary memory consumption in pivot tables

ayuzhin committed:
Fixed bug #49720 (#1948)

SergeyLuzyanin committed:
[bug] Fix bug 50467 (#1949)

konovalovsergey committed:
[all] For bug 50472; Fix export (#1950)

papacarlo committed:
Merge branch release/v6.3.0 into master

GoshaZotov committed:
[se] Fix bug 50489

GoshaZotov committed:
Fix/bug 50509 (#1955)

* [se] Fix bug 50509

GoshaZotov committed:
[se] Fix bug 50521

K0R0L committed:
Fix/critical6.3.1 (#1958)

* [spellcheck] Fix bug 50471

* Fix bug 50482

GoshaZotov committed:
[se] Fix bug 50549

SergeyLuzyanin committed:
[bug] Fix bug 50446

GoshaZotov committed:
[se] By bug 50549

SergeyLuzyanin committed:
[bug] Fix changes error

SergeyLuzyanin committed:
[bug] Fix adding changes to history without point

SergeyLuzyanin committed:
[bug] Fix bug 50607

KhromovNikita committed:
Fixed pasting text into cells of Api.ReplaceTextSmart() method in SpreadSheet editor

KhromovNikita committed:
Changed the used _addChars method to pasteText in Api.ReplaceTextSmart() method.

GoshaZotov committed:
[se] Fix bug 50642

KhromovNikita committed:
Added methods to export (#1967)

KirillovIlya committed:
Fix bug #50604

KirillovIlya committed:
Fix bug #50662

SergeyLuzyanin committed:
[bug] Escape sheet name for regexp on changing sheet name or on deleting sheets

KhromovNikita committed:
Fix bug #50699

SergeyLuzyanin committed:
[bug] Fix losses ole-objects on opening

KhromovNikita committed:
Skip checking hidden columns in Api.ReplaceTextSmart()

SergeyLuzyanin committed:
[bug] Fix bug 50715 (#1984)

GoshaZotov committed:
[se] Fix bug 50717 (#1985)

SergeyLuzyanin committed:
[bug][mobile] fix drawing tracks for chart ranges (#1987)

SergeyLuzyanin committed:
[bug] Fix bug 50757

SergeyLuzyanin committed:
[bug] Fix bug 50762

catbasilio committed:
Merge pull request #1989 from ONLYOFFICE/fix/bug50757

papacarlo committed:
Merge branch hotfix/v6.3.1 into master

Teetoow committed:
Get the latest DocumentServer version
